### PR TITLE
Update config.json 

### LINKED ISSUE #829: queen-attack: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by Akatsuki06**
added topics for the queen-attack problem
fixes #829

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -461,7 +461,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "queen-attack",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "matrices",
+        "conditionals",
+        "mathematics"
```
Comment (FridaTveit) [inline config.json]: These look like great suggestions! :) I would also add `games` as a topic since this is to do with chess :) Do you agree?

Comment (Akatsuki06) [reply config.json]: @FridaTveit 
games and also how about **tuples** ? Can we add even though its not java datatype? or I should specify as **classes** as this specific program contains custom class BoardCoordinate .

Comment (FridaTveit) [reply config.json]: I wouldn't add `tuples` since it's not a java datatype so it could be confusing. But `classes` is a very good suggestion! :)

Comment (Akatsuki06) [reply config.json]: I have done the updates

**Pull request merged**